### ISSUE TITLE
LED configuration in OpenWRT resets color to white

### ISSUE THREAD

**Issue by str-dst** [tnbp/turris-omnia-leds#1]
Hey there :)

First of all really happy that this script exists!  
I found an interesting issue though.

When using the LED configuration in OpenWRT LUCI to change the behaviour of the LEDs they are always reset to white color.  
Changing the color with the script only keeps them at this color until the LED state gets updated by the system.

Would there be any way around this?

**Comment by tnbp** [comment]
This is true and happened to me as well--I added a 4G modem that didn't make the PCI1 LED flash, so I added a rule to at least show its status: it only turned white. (It has started working properly after a recent update, though.)
Right now, I am not quite sure how to go about fixing this. In the meantime, can you tell me which rules you added and why?

**Comment by str-dst** [comment]
I'm using a slightly different setup on the LAN LEDs, instead of showing the state of the individual ports I linked two of the LEDs to  the link-state of the lan-bridge interface (TX and RX separated).  
And I have the PCI2 and 3 LEDs set to show the state of the 2.4 and 5 Ghz wifi networks.

They all have "netdev" set as their trigger, just with different devices linked to them.

Edit:
The config in /etc/config/system for the LEDs looks like this

```
config led
        option name 'pci2'
        option sysfs 'rgb:wlan-2'
        option trigger 'netdev'
        option dev 'phy0-ap0'
        list mode 'link'
        list mode 'tx'
        list mode 'rx'

config led
        option name 'pci3'
        option sysfs 'rgb:wlan-3'
        option trigger 'netdev'
        option dev 'phy1-ap0'
        list mode 'link'
        list mode 'tx'
        list mode 'rx'

config led
        option name 'LAN 2 TX'
        option sysfs 'rgb:lan-2'
        option trigger 'netdev'
        option dev 'br-lan'
        list mode 'link'
        list mode 'tx'

config led
        option name 'LAN 2 RX'
        option sysfs 'rgb:lan-3'
        option trigger 'netdev'
        option dev 'br-lan'
        list mode 'link'
        list mode 'rx'
```

**Comment by tnbp** [comment]
I am not ignoring you, I'm just flummoxed right now. I don't know how to fix this.
If I find out, I'll reply though.

**Comment by str-dst** [comment]
Ay no worries haha 😄 

Big thanks for looking into this!

**Comment by tnbp** [comment]
So I've tried to reproduce this issue, but I couldn't: my LEDs stay the color I set it, no matter what rules I set.
Can you tell me which openWRT version you're using?
